pkg/lib/cockpit.d.ts

```diff
@@ -131,8 +131,17 @@ declare module 'cockpit' {
         close(): void;
     }
 
+    type VariantType = string | string[] | Uint8Array | number | number[] | boolean | boolean[];
```

Comment:
That looks vaguely like `cockpit.JsonValue`...

Reply:
Hmmm but is it? :)